godtossing/Pos/Ssc/GodTossing/Configuration.hs

```diff
@@ -0,0 +1,21 @@
+{-# LANGUAGE Rank2Types #-}
+
+module Pos.Ssc.GodTossing.Configuration
+    ( GtConfiguration
+    , HasGtConfiguration
+    , gtConfiguration
+    , withGtConfiguration
+    ) where
+
+import           Pos.Ssc.GodTossing.Genesis.Types
+import           Data.Reflection                  (Given (..), give)
+
+type HasGtConfiguration = Given GtConfiguration
+
+type GtConfiguration = GenesisGtData
```

Comment:
It will become unneeded soon, because `GenesisGtData` will be removed. We moved `VssCertificatesMap` into `cardano-sl-core`, so now this `GenesisGtData` is either part of `GenesisSpec` or can be uniquely generated from it.

Reply:
Could it be that we'll need the configuration in order to produce the configuration? The protocol magic number is not known until we get the genesis spec, but it's needed  in order to make `VssCertificate`.

Reply:
It might be possible to define a lazy version of `Data.Reflection.give`... `giveFix :: forall a r. (a -> a) -> (Given a => r) -> r`

Reply:
No, `GenesisSpec` is exactly what is stored in a yaml file. Obtaining `GenesisSpec` is simple `Data.Yaml.decodeEither`. So there shouldn't be a problem.
In some cases it stores `VssCertificate`, but in this case we just put it in some format (determined `ToJSON` instances) into yaml file.